cmd/trace-agent/config/config.go

```diff
@@ -222,6 +221,16 @@ func applyDatadogConfig(c *config.AgentConfig) error {
 	if coreconfig.Datadog.IsSet("apm_config.max_remote_traces_per_second") {
 		c.MaxRemoteTPS = coreconfig.Datadog.GetFloat64("apm_config.max_remote_traces_per_second")
 	}
+	if k := "apm_config.features"; coreconfig.Datadog.IsSet(k) {
+		tempFeats := coreconfig.Datadog.GetStringSlice(k)
+		for _, feat := range tempFeats {
+			c.Features[feat] = struct{}{}
+		}
```

Comment:
I can't find the source, but in general with Go, you give variables longer and more descriptive names the farther from the original context they carry. So in this case, `tempFeats` is only declared then used once in the following line, so it can be quite short. Same with `feat` in the loop. Since the context they can be used is so small, the names can be short as well (while still being sufficiently descriptive)

Reply:
lol I think @mtoffl01 asked me about this exact piece of code if these names were ok when we were pairing on this and I said they were fine. :P (Also I might use `feats` instead of just `feat` personally since it makes it more obvious it's more than one)

Reply:
agree, @ajgajg1134 . Implementing katie's suggestion, but with the plural